peer/x/roundrobin/list.go

```diff
@@ -120,20 +121,15 @@ func (pl *List) addToAvailablePeers(p peer.Peer) error {
 
 // Start notifies the List that requests will start coming
 func (pl *List) Start() error {
-	if pl.started.Swap(true) {
-		return peer.ErrPeerListAlreadyStarted("RoundRobinList")
-	}
-	return nil
+	return pl.startOnce.Do(func() error { return nil })
```

Comment:
I'll change it back to the repetitive way for now, of course that will have the issue of copy-pastaism, but since it's hidden behind the api at least we can change it.